build-managed.cmd

```diff
@@ -0,0 +1,2 @@
+@call %~dp0run.cmd build-managed -nodoReuse -binclashWindows %*
```

Comment:
Do we have documentation for the build-managed command?  The docs only refer to build to my knowledge at this point.